text/0000-const-trait-impls.md

```diff
@@ -103,84 +109,117 @@ So, we need some annotation that differentiates a `T: Default` bound from one th
 
 ## Nomenclature and new syntax concepts
 
-### Const trait impls
+### Const trait methods
 
-It is now allowed to prefix a trait name in an impl block with `const`, marking that this `impl`'s type is now allowed to
-have methods of this `impl`'s trait to be called in const contexts (if all where bounds hold, like usual, but more on this later).
+Traits can declare methods as `const`. Doing so is a breaking change, as all impls are now required to provide a `const` method,
```

Comment:
*adding* const to an existing method is a breaking change.

Reply:
the point is adding a new defaulted function is fine to be `const`. it's when you add `const` to an existing function